docs/api/references.rst

```diff
@@ -137,6 +137,14 @@ References
            of the Air Force Global Weather Central*. `AD0744042
            <https://apps.dtic.mil/sti/citations/AD0744042>`_.
 
+.. [MillerBidnerMaddox1971] Miller, R. C, A. Bidner, and R. A. Maddox, 1971: *The use of
+           computer products in severe weather forecasting (the SWEAT index)*. Preprints.
+           Seventh Conf. Severe Local Storms. Kansas City, Amer. Meteor. Soc., 1-6.
+
+.. [MillerBidnerMaddox1972] Miller, R. C, A. Bidner, and R. A. Maddox, 1972: *The use of
+           computer products in severe weather forecasting (the SWEAT index)*. Air Weather
+           Service Aerospace Sciences Review. AWS RP 105-2, No. 72-1, 2-9.
+
```

Comment:
Instead of adding these, let's keep things simpler by just citing Miller1972, which covers the entire thing in Appendix F.